doofinder.php

```diff
@@ -1354,6 +1390,8 @@ public function setSearchEnginesByConfig()
                 Configuration::updateValue('DF_HASHID_' . strtoupper($currency) . '_' . strtoupper($lang), $hashid);
             }
         }
+
+        return true;
```

Comment:
This function is called on module update. Functions that are called on install or update need to return a boolean so that the module knows that the update was successful.